processing-for-variant-discovery-gatk3.wdl

```diff
@@ -429,13 +442,19 @@ workflow GenericPreProcessingWorkflow {
   Int agg_medium_disk
   Int agg_large_disk
 
+  String? gitc_docker
+  String gitc_image = select_first([gitc_docker, "broadinstitute/genomes-in-the-cloud:2.3.1-1512499786"])
```

Comment:
I'm unhappy with the variable names `gitc_docker` vs. `gitc_image`...not that I have a better suggestion. Perhaps `default_image` and `docker_image`?

Reply:
The "gitc" is specified in the variable because there times when a wdl would have more than one docker image e.g. gatk, oncotator. So normally the docker image variables have distinct names in a wdl. This particular wdl uses one docker image so I'll make the change if you prefer.

Reply:
it's not so much the "gitc" part as the "docker" versus "image" that doesn't really make sense to me...